services/core-web/src/components/dashboard/majorProjectHomePage/MajorProjectTable.tsx

```diff
@@ -0,0 +1,227 @@
+    key: section.key,
+    tab: section.tab,
+    projectGuid: project.key,
+    status: statusCodes[section.status_code] || statusCodes["NTS"],
+    updateTimestamp: formatDate(section.update_timestamp) || Strings.EMPTY_FIELD,
+    updateUser: section.update_user,
+  }
+}
+
+const handleTableChange = (handleSearch, tableFilters) => (pagination, filters, sorter) => {
+  const params = {
+    ...tableFilters,
+    sort_field: sorter.order ? sorter.field : undefined,
+    sort_dir: sorter.order ? sorter.order.replace("end", "") : undefined,
+  };
+  handleSearch(params);
+};
+
+export const MajorProjectTable: FC<MajorProjectsSearchFormProps> = ({
+  projects,
+  mineCommodityOptionsHash,
+  handleSearch,
+  filters,
+  isLoaded,
+  sortField,
+  sortDir,
+  expandedRowKeys,
+  onExpand,
+}) => {
+  const { isFeatureEnabled } = useFeatureFlag();
+  const projectDecisionPackageEnabled = isFeatureEnabled(Feature.MAJOR_PROJECT_DECISION_PACKAGE);
+  const columns = [
+    {
+      key: "project_title",
+      title: "Project Title",
+      dataIndex: "project_title",
+      sortField: "project_title",
+      sorter: true,
+      render: (text, record) => (
+        <Link to={router.EDIT_PROJECT.dynamicRoute(record.key)} title="Name">
+          {text}
+        </Link>
+      ),
+    },
+    renderTextColumn("project_id", "Client ID", true),
+    renderTextColumn("mine_name", "Mine Name", true),
+    renderTextColumn("project_lead_name", "EMLI Project Lead", true),
+    {
+      key: "commodity",
+      title: "Commodity",
+      dataIndex: "commodity",
+      sortField: "commodity",
+      sorter: false,
+      render: (text) => (
+        <div title="Commodity">
+          {(text && text.length > 0 && text.join(", ")) || Strings.EMPTY_FIELD}
+        </div>
+      ),
+    },
+    {
+      title: "Updated Date",
+      key: "update_timestamp",
+      dataIndex: "update_timestamp",
+      sortField: "update_timestamp",
+      render: (text) => <div title="Updated Date">{text}</div>,
+      sorter: dateSorter("update_timestamp"),
+      defaultSortOrder: "descend",
+    },
+    {
+      title: "",
+      key: "project",
+      dataIndex: "project",
+      render: (text, record) => (
+        // @ts-ignore
+        <div title="" align="right" data-cy="major-projects-table-open-button">
+          <Row gutter={1}>
+            <Col span={12}>
+              <Link to={router.EDIT_PROJECT.dynamicRoute(record.key)}>
+                <Button type="primary">Open</Button>
+              </Link>
+            </Col>
+          </Row>
+        </div>
+      ),
+    },
+  ];
+
+  const childColumns = [
+    { ...renderTextColumn("key", "Project Stage") },
+    { ...renderTextColumn("status", "EMLI Review Status") },
+    { ...renderTextColumn("updateTimestamp", "Last Updated") },
+    { ...renderTextColumn("updateUser", "Updated By") },
+    {
+      title: "",
+      key: "project_section",
+      dataIndex: "project_section",
+      render: (text, record) => {
```

Comment:
Looks like the render method doesn't use the `text` parameter (which will be `record.project_section`). If that's the case- you can take out the dataIndex and the text parameter completely.

Reply:
Thanks this has been removed now.